exporter/datadogexporter/translate_traces.go

```diff
@@ -487,7 +487,14 @@ func decodeAPMId(id string) uint64 {
 	return val
 }
 
-func getDatadogSpanName(s pdata.Span, datadogTags map[string]string) string {
+func getDatadogSpanName(s pdata.Span, datadogTags map[string]string, spanNameAsResourceName bool) string {
+	// Option created to maintain similarity with the OpenTelemetry semantic conventions
```

Comment:
This text already exists in the `yaml`. At most, add it once more in `exporter/datadogexporter/config/config.go`, but I don't think it belongs here.